Align actuals timeseries with the projections timeseries.

mikelehen opened the pull request.

goldblatt commented on src/models/Projection.ts:
```diff
@@ -166,12 +172,12 @@ export class Projection {
+        this.actualTimeseries.map(row => row && row.ICUBeds.currentUsageCovid),
       );
-      return latestCurrentUssage - latestUsageCovid;
+      return latestCurrentUssage! - latestUsageCovid!;
```
lol

goldblatt commented on src/models/Projection.ts:
```diff
@@ -229,12 +235,29 @@ export class Projection {
+
+    const firstDate = moment(this.dates[0]);
+    const firstActualDate = moment(actuals[0].date);
+    const padFrontDays = firstActualDate.diff(firstDate, 'days');
```
is the data daily for sure now?

goldblatt replied:
otherwise the dates might not line up is what i'm thinking jst seems tenuous

mikelehen commented:
I reworked this completely to basically walk the two timeseries in parallel and match trim / pad / fill in missing dates in the actuals.  I added a bunch of asserts and verified that the /all page doesn't crash, so I think it probably is working as intended. :-)

But I don't like this approach, so feel free to come up with something better.
